src/vonage/ncco_builder/pay_prompts.py

```diff
@@ -1,12 +1,12 @@
-from pydantic import BaseModel, validator
-from typing import Optional, Dict
+from pydantic import BaseModel, ValidationInfo, field_validator, validator
+from typing import Dict
 from typing_extensions import Literal
 
 
 class PayPrompts:
     class VoicePrompt(BaseModel):
-        language: Optional[str]
-        style: Optional[int]
+        language: str
+        style: int
```

Comment:
```suggestion
        style: int = None
```

Reply:
I also tabbed through - only found one other place where the None was missing